From 7dc11e25714e01cf73186875ff0057767fa09e18 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Mon, 2 Mar 2015 15:43:28 +0800
Subject: [PATCH] MLK-10373-1 ARM: dtsi: imx7d: add dts support for the lcdif
 controller

Add the required dts entries for the lcdif controller
on IMX7D board.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
Signed-off-by: Fancy Fang <chen.fang@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 8614fcb..55179d7 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -717,11 +717,11 @@
 				status = "disabled";
 			};
 
-			lcdif1: lcdif@3073000 {
+			lcdif: lcdif@30730000 {
 				compatible = "fsl,imx7d-lcdif", "fsl,imx28-lcdif";
-				reg = <0x3073000 0x10000>;
+				reg = <0x30730000 0x10000>;
 				interrupts = <GIC_SPI 5 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX7D_CLK_DUMMY>,
+				clocks = <&clks IMX7D_LCDIF_PIXEL_ROOT_CLK>,
 					<&clks IMX7D_CLK_DUMMY>,
 					<&clks IMX7D_CLK_DUMMY>;
 				clock-names = "pix", "axi", "disp_axi";
-- 
1.8.0

